app/main.py

```diff
@@ -1,8 +1,69 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: int | float,
+            clean_power: int,
+            average_rating: int | float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        """
+        The method, that takes a list of cars, washes only cars with clean_mark < clean_power of
+        wash station and returns income of CarWashStation for serving this list of cars,
+        rounded to 1 decimal.
+        """
+        total_income = 0.0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                total_income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(total_income, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        """
+        The method, that calculates cost for a single car wash, cost is calculated as: car's comfort class
+        * difference between wash station's clean power and car's clean mark
+        * car wash station rating / car wash station distance to the center of the city.
+        """
+        price = (
```

Comment:
Code Style Issue: The calculation of the price is a bit hard to read due to the complexity of the calculation. Consider splitting it into multiple variables or breaking after each binary operation for better readability. This is in line with the first point in the Code Style section of the checklist.